docsbox/docs/views.py

```diff
@@ -64,7 +62,7 @@ def post(self, file_id):
             if request.files and "file" in request.files:
                 mimetype = get_file_mimetype_from_data(request.files["file"])
             elif file_id and is_valid_uuid(file_id):
-                mimetype = get_file_info(request, file_id)["mimetype"]
+                mimetype = get_file_info(file_id, request.headers.get('Content-Disposition'))["mimetype"]
```

Comment:
Does the headers.get() return None or raise an error if the header is not present?

Reply:
headers is a dictionary, the get method returns None if the key does not exist and its not given a default value